changelogs/client_server/newsfragments/1994.feature

```diff
@@ -0,0 +1 @@
+Remove reply fallbacks, as per [MSC2781](https://github.com/matrix-org/matrix-spec-proposals/issues/2781).
```

Comment:
It should probably be a `*.removal` changelog, but given that the generated heading says "Endpoints Removed", it is not quite right.

Reply:
I personally am totally fine to consider it a feature :)